bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,257 @@
+* Deutsche Telekom Media Receiver MR601  - should be supported (not verified)
+* Deutsche Telekom Media Receiver MR3xx  - NOT supported (different platform)
+* Deutsche Telekom Media Receiver MR1xx  - NOT supported (different platform)
+
+## Auto Discovery
+
+UPnP will be used to discover receivers on the local network and discover the nessesary parameters.
+The receiver needs to be powered on to get discovered.
+
+Once the receiver is discovered it can be added from the Inbox.
+Make sure to set the parameter `userId` in the Thing configuration after adding the new thing, see section Thing Configuration.
+
+Note:
+The binding uses the network settings in openHAB's system configuration to determine the local IP address.
+The device can't be discovered if the openHAB system and receiver are not on the same network (IP/Netmask).
+In this case you need to add the Thing manually or use textual configuration (.things).
+
+If you are running openHAB in a Docker container you need to make sure that UPnP discovery is available and network interfaces 
+
+## Generating the UID (`userId`)
+
+The binding requires a so called UID (`userId`) to pair with the receiver, which is generated by logging in with your Telekom credentials (Login Name + Password for the "Telekom Kundencenter").
+openHAB needs to access the Internet to perform that operation, which can be disabled afterwards.
+
+The binding initiates the pairing with the receiver once the userId has been obtained, this is an automated process.
+The Thing changes to ONLINE once the pairing result is received, otherwise check the Thing status for an error message.
+
+There are different ways to setup the userId:
+
+1. Use the openHAB console
+Run the following command on the console and provide your Tesla account credentials (the same that you use in the official Tesla app):
+
+```
+openhab> smarthome:magentatv login
+Username (email): mail@example.com
+Password: topsecret
+
+Attempting login...
+Login successful, returned UID (userId) is 1903AAAAAAAAC7E9718BBBCBCBCBCBC
+Edit thing configuration and copy this value to the field UID
+```
+
+On successful login the console will show the userId value. Copy&amp;Paste this value to the Thing configuration (parameter UID / `userId`) of the receiver.
+If you have multiple receivers under the same MagentaTV subscription you could re-use this value for the other receiver Things. 
+
+2. Provide your credentials in the UI
+If you do not want to use the openHAB console, you can also setup the credentials in the Thing configuration 
+
+- Account Name (`accountName`) is your Login Name for the Telekom Kundencenter (registered email address) 
+- Account Password (`accountPassword`) is the corresponding password.
+
+The binding will do an automated login, receives the UID (`userId`) and stores this in the Thing configuration.
+For security reasons the credentials will be automatically deleted from the thing configuration (replaced with '***') after the initial process.
```

Comment:
```suggestion
For security reasons the credentials are automatically deleted from the thing configuration (replaced with '***') after the initial process.
```
Err, why replaced with '***'? The two parameters should be set to null instead.

Reply:
done